pkg/logs/launchers/integration/launcher.go

```diff
@@ -157,56 +253,117 @@ func (s *Launcher) makeFileSource(source *sources.LogSource, logFilePath string)
 
 // TODO Change file naming to reflect ID once logs from go interfaces gets merged.
 // createFile creates a file for the logsource
-func (s *Launcher) createFile(id string) (string, error) {
-	logFilePath := s.integrationLogFilePath(id)
+func (s *Launcher) createFile(source string) (*FileInfo, error) {
+	filepath := s.integrationLogFilePath(source)
 
-	file, err := os.Create(logFilePath)
+	file, err := os.Create(filepath)
 	if err != nil {
-		return "", nil
+		return nil, nil
 	}
+	ddLog.Info("Successfully created integrations log file.")
```

Comment:
should we print the path here so this log line is more useuful?

Reply:
Added! 👍

Reply:
The log line is still missing the path here

Reply:
I think some things were lost when I merged main (and didn't handle the merge conflicts correctly 😅, I've gone back through all the comments and made sure the changes are now reflected in the code.